slugline/urls.py

```diff
@@ -15,17 +15,46 @@
 """
 from django.conf import settings
 from django.conf.urls.static import static
+from django.conf.urls import url
 from django.contrib import admin
 from django.contrib.auth.views import LogoutView
 from django.urls import path, include
 from django.shortcuts import render
 
+from rest_framework import permissions
+from drf_yasg.views import get_schema_view
+from drf_yasg import openapi
+
 import content.urls
 import content.views
 import user.urls
 import user.views
 
+swagger_info = openapi.Info(
+    title="goosePRESS API",
+    default_version="v1",
+    description="API for goosePRESS",
+    # terms_of_service="https://www.google.com/policies/terms/",
+    # contact=openapi.Contact(email="contact@snippets.local"),
```

Comment:
Should this be changed to our contact email?

Reply:
I'll take another look once we have the license and everything done.